include/crypto/bn.h

```diff
@@ -116,4 +116,13 @@ OSSL_LIB_CTX *ossl_bn_get_libctx(BN_CTX *ctx);
 
 extern const BIGNUM ossl_bn_inv_sqrt_2;
 
+#if defined(OPENSSL_SYS_LINUX) && !defined(FIPS_MODULE) && defined (__s390x__)
+#  define S390X_MOD_EXP
```

Comment:
This should be indented just by one space.

Reply:
Fixed.